store/rootmulti/store.go

```diff
@@ -598,9 +601,11 @@ func (rs *Store) PruneStores(clearPruningManager bool, pruningHeights []int64) (
 		return nil
 	}
 
-	rs.logger.Debug("pruning heights", "heights", pruningHeights)
+	rs.logger.Debug("prune store", "heights", pruningHeights)
 
 	for key, store := range rs.stores {
+		rs.logger.Debug("prune store", "key", key)
```

Comment:
```suggestion
		rs.logger.Debug("pruning store", "key", key, "store", store.Name)
```

Reply:
store.name is private.  Should we make it public?